### DISCUSSION TITLE
Vehicle slides down slope when stationary

### DISCUSSION THREAD

**Discussion by farbod1277** [NVIDIA-Omniverse/PhysX#94] [Q&A] [answered]
Hi there,

I'm using the SnippetVehicleDirectDrive, and when I change the infinite ground plane to have even a small angle of 5 degrees in the longitudinal direction of the vehicle. The vehicle is unable to stay stationary even when the brakes are applied. It slides (not roll, the wheels are not spinning) down the slope. What do I need to change to make sure that this doesn't happen?

Additionally if I set the ground plane to have a small slope in the lateral direction the vehicle slides down laterally but not continuously, it just slides for a bit then stops and slides again.

Thanks

**Comment by preist-nvidia** [comment, marked as answer]
Converted to Q&A.

**Reply by msauter-nvidia** [reply]
Hi,
You could try to tune the parameters of the PxVehicleTireAxisStickyParams struct a bit (especially thresholdSpeed) and see if that helps. They exist for both longitudinal and lateral direction (see PxVehicleTireStickyParams).
Also make sure PxVehicleCommandState.throttle is 0.

**Reply by farbod1277** [reply]
Hi @msauter-nvidia , thanks for your response. I set the thersholdSpeed to zero and it helped.
